WeatherApp/WeatherAppTests/Controllers/AutocompleteControllerTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NSubstitute;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WeatherApp.WebSite.Controllers;
+using WeatherApp.WebSite.Models;
+using WeatherApp.WebSite.Services;
+
+namespace WeatherApp.WebSite
+{
+    [TestFixture]
+    public class AutocompleteTests
```

Comment:
Negative cases should be covered with tests also.

Reply:
Thanks for the remark, one more case added for now.